sdk/java/src/main/java/com/chain/api/Transaction.java

```diff
@@ -875,8 +877,50 @@ public ControlWithAccount setAmount(long amount) {
     }
 
     /**
+     * Use this action to pay assets into a {@link Receiver}.
+     */
+    public static class ControlWithReceiver extends Action {
+      /**
+       * Default constructor.
+       */
+      public ControlWithReceiver() {
+        this.put("type", "control_with_receiver");
+      }
+
+      /**
+       * Specifies the receiver that is being paid to.
+       *
+       * @param receiver the receiver being paid to
+       * @return the updated action object
+       */
+      public ControlWithReceiver setReceiver(Receiver receiver) {
+        this.put("receiver", receiver);
+        return this;
+      }
+
+      /**
+       * Specifies the receiver (as a JSON string) that is being paid to.
+       *
+       * @param serialized the JSON-serialized receiver being paid to
+       * @return the updated action object
+       */
+      public ControlWithReceiver setReceiver(String serialized) throws JSONException {
```

Comment:
Maybe we should throw a more generic exception than a `JSONException` to preserve the flexibility to serialize receivers in other ways (ex, protocol buffers) in the future.

On the same subject, would a different type for `serialized` allow more flexibility for binary serialization formats?